addon/components/frost-list.js

```diff
@@ -114,6 +136,36 @@ export default Component.extend({
     })
   },
 
+  @readOnly
+  @computed('componentKeyNamesForTypes')
+  isAnyTypedItemExpansion (componentKeyNamesForTypes) {
+    const componentKeyNames = this.get('componentKeyNames')
+    const itemExpansionKeyName = get(componentKeyNames, 'itemExpansion')
+
+    for (var itemType in componentKeyNamesForTypes) {
```

Comment:
use let instead of var

Reply:
This was inserted by my editor from a snippet, I will keep a look out for this.